en/lessons/basics/basics.md

```diff
@@ -101,6 +101,7 @@ false
 ```
 
 The booleans `true` and `false` are also the atoms `:true` and `:false`, respectively.
+*The pipe operator |> passes the result of an expression as the first parameter of another expression. We will explore this more in-depth in chapter 7: "Pipe Operator"*
```

Comment:
I'm wondering if we should change these to `is_atom(true)` and `is_atom(false)` since we often discourage using pipes for a single function calls anyways and instead add a small section on the pipe operator in this lesson.

What do you think @Snackomat?

Reply:
I think that would really help newcomers aswell, because (atleast for me) that would've clarified it way more and also suits the way I expect a language to behave.

Reply:
@snackomat we'll get this merged in and then if you're up for it, I'd love for you to submit a PR for the pipe section 😁